openeo-geotrellis/src/test/scala/org/openeo/geotrellis/OpenEOProcessesSpec.scala

```diff
@@ -291,6 +298,71 @@ class OpenEOProcessesSpec extends RasterMatchers {
 
   }
 
+  @Test
+  def aggregateTemporalTest(): Unit = {
+    println("Test if this test runs in CI.")
+    val outDir = "/tmp/aggregateTemporalTest/"
+    Files.createDirectories(Paths.get(outDir))
+    List(
+      DoublePixelType(),
+      FloatPixelType(),
+      IntPixelType(),
+      ShortPixelType(),
+      BytePixelType(),
+      BitPixelType()
+    ).foreach(pixelType => {
```

Comment:
a for loop like this can be turned into a parametrized test, which gives more convenient output

Reply:
Used `@ParameterizedTest` way.